### PR TITLE
docs: add two blogs

### PR DESCRIPTION
**Pull request by EstelleRao**
Add two contributor interview blogs both in Chinese and English

### CODE REVIEW

File: website/blog/2022/04/29/interview-juzhiyuan-apache-member.md [hunk 1]
```diff
@@ -0,0 +1,167 @@
...
+- ASF Member
+- Apache Member
+- Open Source
+- Remote Work
+- Full-Stack Web Developer
+description: In this interview, we invited JU Zhiyuan, a PMC member of Apache APISIX, to share his feeling about becoming an ASF member, his experience with open source, and his self-learning journey in technology.
+tags: [Interview]
+---
+
+>JU Zhiyuan, a PPMC member of Apache APISIX, was elected to be one of the 918 ASF Members in March 2022. In this interview, Zhiyuan shared his experience with open source and his self-learning journey in technology.
```
Comment (yzeng25) [inline website/blog/2022/04/29/interview-juzhiyuan-apache-member.md]: Need a space here.
```suggestion
> JU Zhiyuan, a PPMC member of Apache APISIX, was elected to be one of the 918 ASF Members in March 2022. In this interview, Zhiyuan shared his experience with open source and his self-learning journey in technology.

```

File: website/blog/2022/04/29/interview-juzhiyuan-apache-member.md [hunk 2]
```diff
@@ -0,0 +1,167 @@
...
+
+**What is open source? In the video named [Trillions and Trillions Served](https://www.youtube.com/watch?v=JUt2nb0mgwg), Jim Jagielski, one of the co-founders of the Apache Software Foundation, pointed out that "for me, it's an acknowledgment of what I see was the old hacker culture. I mean that developers and coders contributors are artists, and I think all artists want to share the results of their craft. They want to work with aligned people. They want to hone their skills."  
+You got to know hackers early in your junior and high school and then engaged in open-source communities in college. How do you take the spirit of hackers and open source? In your opinion, is there any inheritance between hacker culture and open-source culture?**
```
Comment (yzeng25) [inline website/blog/2022/04/29/interview-juzhiyuan-apache-member.md]:
```suggestion
**What is open source? In the video named [Trillions and Trillions Served](https://www.youtube.com/watch?v=JUt2nb0mgwg), Jim Jagielski, one of the co-founders of the Apache Software Foundation, pointed out that "for me, it's an acknowledgment of what I see was the old hacker culture. I mean that developers and coders contributors are artists, and I think all artists want to share the results of their craft. They want to work with aligned people. They want to hone their skills."

**You got to know hackers early in your junior and high school and then engaged in open-source communities in college. How do you take the spirit of hackers and open source? In your opinion, is there any inheritance between hacker culture and open-source culture?**
```

File: website/i18n/zh/docusaurus-plugin-content-blog/2022/04/29/interview-juzhiyuan-apache-member.md [hunk 3]
```diff
@@ -0,0 +1,174 @@
...
+
+因为 freeCodeCamp 是开源项目，连续多年都是 GitHub Top 1 的项目，有几百 K 的 Star，我会改 Bug，提交 PR，**很有成就感**。通过 freeCodeCamp 平台，我们也**吸引了很多技术爱好者，帮助他们从 0 到 1 学习技术**。在这个过程中，我也**形成了自己的做事方式**，不管是教人学习 Web 还是带人 On Boarding，都**沉淀了一些可持续的指导经验**，这也是我后面敢带其他人的原因。带他们上岸后，我会继续引导他们去带更多人。这是一个**良性循环**的过程。虽然做这些事情付出了很多时间，看似没有任何直接的物质收益，但却给我带来了很多精神上的收益和潜在收益，比如个人影响力有慢慢地提升，收获的感悟和形成的做事方式还在影响着我的现在和未来。
+
+在大学期间，我持续参与了各种社区和技术分享，学习技术，分享技术，实践项目。因为热爱，我坚定走在 Web 技术学习这条路上，同时也不忘鼓励身边的朋友一起学习。我们会在宿舍、在学校机房举办计算机相关的活动，一起做项目，也会去其他城市的学校给同学们做分享。几年下来，我积攒了很多项目经验，拓展了技术栈，但是我发现重复做外包项目虽然可以带来暂时性收入，但不利于个人技术的成长。我意识到在开发过程中使用的开源库代码质量高，协作流程规范，我便开始学习开源项目的代码，关注开源社区的动态。毕业后，我接触到了 APISIX 开源项目，我就参与进来了。
+
+![Zhiyuan's Resume Written in 2018](https://static.apiseven.com/202108/1651202016764-e461af74-7aca-466d-8824-e823195fc24f.png)
+
+**Apache APISIX 社区：在 [Trillions and Trillions Served](https://www.youtube.com/watch?v=JUt2nb0mgwg) 视频里，Apache 软件基金会的创始成员 Jim Jagielski 在分享“什么是开源”时提到，“于我而言，参与开源是出于对古老的黑客文化的认同。开发者、贡献者都是艺术家。在我看来，所有的艺术家都乐于分享成果，喜于和志同道合的人协作，专注于磨练技能。”**
+**你初高中接触到黑客群体，大学接触到开源社区，你如何理解黑客精神、开源精神？在你看来，黑客文化和开源文化之间有何承继关系或有何渊源？**
```
Comment (yzeng25) [inline website/i18n/zh/docusaurus-plugin-content-blog/2022/04/29/interview-juzhiyuan-apache-member.md]:
```suggestion
**Apache APISIX 社区：在 [Trillions and Trillions Served](https://www.youtube.com/watch?v=JUt2nb0mgwg) 视频里，Apache 软件基金会的创始成员 Jim Jagielski 在分享“什么是开源”时提到，“于我而言，参与开源是出于对古老的黑客文化的认同。开发者、贡献者都是艺术家。在我看来，所有的艺术家都乐于分享成果，喜于和志同道合的人协作，专注于磨练技能。”**

**你初高中接触到黑客群体，大学接触到开源社区，你如何理解黑客精神、开源精神？在你看来，黑客文化和开源文化之间有何承继关系或有何渊源？**

```

**Comment by yzeng25** [comment]
This pull request is a newer version of https://github.com/apache/apisix-website/pull/1064, which has been closed due to too much changes in a short period of time.

I find some neats needed to be fixed, others LGTM.

@hf400159 @SylviaBABY @juzhiyuan please take a look when available, thanks!

File: website/blog/2022/04/29/interview-juzhiyuan-apache-member.md [hunk 4]
```diff
@@ -0,0 +1,167 @@
...
+
+When I was in college, I also played drones, took aerial photos of the school with my friends, and then reproduced them into a video with Minecraft, which is available on Bilibili ([a promotional video of Henan Normal University](https://www.bilibili.com/video/BV1As411a7Mc?from=search&seid=7847808159839607455&spm_id_from=333.337.0.0)). We also used drones to shoot sunrise on the top of Mount Tai, help the hometown shoot various videos, and help schools broadcast various activities live.
+
+## Knowing More About Zhiyuan
+
+Zhiyuan usually shares his recent thoughts and insights in his blogs. Through his words, we can see a young man exploring freely in open source, technology, work, and life. If you want to know more about Zhiyuan, you can subscribe to his [Blog](https://wineso.me/) and [GitHub](https://github.com/juzhiyuan).
+Looking forward to more exploration and sharing from Zhiyuan in the future!
```
Comment (SylviaBABY) [inline website/blog/2022/04/29/interview-juzhiyuan-apache-member.md]:
```suggestion
Zhiyuan usually shares his recent thoughts and insights in his blogs. Through his words, we can see a young man exploring freely in open source, technology, work, and life. If you want to know more about Zhiyuan, you can subscribe to his [Blog](https://wineso.me/) and [GitHub](https://github.com/juzhiyuan).

Looking forward to more exploration and sharing from Zhiyuan in the future!

```

**Comment by SylviaBABY** [comment]
No big problem, mainly some paragraphs with empty lines. Others LGTM

File: website/i18n/zh/docusaurus-plugin-content-blog/2022/04/29/interview-juzhiyuan-apache-member.md [hunk 5]
```diff
@@ -0,0 +1,174 @@
...
+
+**Apache APISIX 社区：除了工作以外，平时生活中会关注哪些领域的东西呢？**
+
+**致远**：我把当前的工作看成一个事业，也是一个兴趣爱好。而且创业本身很有趣，我现在负责公司的 Global Team，团队内有很多外国同事。
+
+我也会关心教育公益，比如参与或组织 freeCodeCamp 的活动，鼓励并帮助身边的人学习计算机、学习技术。我认为**自己成长的前提是身边的人一起成长**。能帮助他们通过自己的手艺实现自己的想法，这是一件很酷的事情。虽然我现在个人精力有限，但是我会鼓励我 Mentor 过的同学带新人，帮助新人学习 Web 技术。
+
+有时我也在想，如果我不干这行，我会做什么。我的博客签名是「**用魔法打败魔法**」。我觉得化学是一种很神奇的东西，小时候参加过巴斯夫小小化学家活动，知道著名化工企业巴斯夫在用化学改变生活。我对自然、生物也比较感兴趣，看过很多宇宙、自然相关的百科全书。  
+
+大学期间，我还玩过无人机，和小伙伴们一起航拍学校，再用 Minecraft 复刻成视频，Bilibili 上有这个视频（[河南师范大学宣传视频](https://www.bilibili.com/video/BV1As411a7Mc?from=search&seid=7847808159839607455&spm_id_from=333.337.0.0)）。我们也会拎着无人机在泰山山顶拍日出，帮家乡拍各种视频，帮学校直播各种活动。
```
Comment (hf400159) [inline website/i18n/zh/docusaurus-plugin-content-blog/2022/04/29/interview-juzhiyuan-apache-member.md:168]:
```suggestion
大学期间，我还玩过无人机，和小伙伴们一起航拍学校，再用 Minecraft 复刻成视频，Bilibili 上有这个视频（[河南师范大学宣传视频](https://www.bilibili.com/video/BV1As411a7Mc?from=search&seid=7847808159839607455&spm_id_from=333.337.0.0)。我们也会拎着无人机在泰山山顶拍日出，帮家乡拍各种视频，帮学校直播各种活动。

```

**Review by guitu168 — COMMENTED** [review]
LGTM

File: website/blog/2022/04/29/interview-juzhiyuan-apache-member.md (hunk 4 above)
Comment (EstelleRao) [reply website/blog/2022/04/29/interview-juzhiyuan-apache-member.md]: Empty line added

File: website/i18n/zh/docusaurus-plugin-content-blog/2022/04/29/interview-juzhiyuan-apache-member.md (hunk 3 above)
Comment (EstelleRao) [reply website/i18n/zh/docusaurus-plugin-content-blog/2022/04/29/interview-juzhiyuan-apache-member.md]: Fixed markdown styling problems

File: website/blog/2022/04/29/interview-juzhiyuan-apache-member.md (hunk 1 above)
Comment (EstelleRao) [reply website/blog/2022/04/29/interview-juzhiyuan-apache-member.md]: Space added

**Review by SylviaBABY — APPROVED** [review]

**Review by yzeng25 — APPROVED** [review]
LGTM

**Review by guitu168 — APPROVED** [review]
LGTM

**Pull request merged**